src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -725,9 +728,17 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
                 }
                 return Task.FromResult<object>(null);
             });
+
+            var console = (Func<object, Task<object>>)(msg =>
+            {
+                _consoleLogger?.LogDebug((string)msg);
```

Comment:
Since you're not also logging this to a TraceWriter, we won't have the SystemTraceWriter wired up, which means we won't see these from the backend logs. I suppose that's by design?